cfme/tests/services/test_cloud_service_catalogs.py

```diff
@@ -36,27 +39,31 @@ def test_cloud_catalog_item(appliance, setup_provider, provider, dialog, catalog
     provisioning_data = {
         'catalog': {'vm_name': vm_name,
                     },
-        'properties': {'instance_type': provisioning['instance_type'],
+        'properties': {'instance_type': partial_match(provisioning['instance_type']),
                        },
-        'environment': {'security_groups': [provisioning['security_group']],
-                        },
+        'environment': {'security_groups': partial_match(provisioning['security_group']),
```

Comment:
I think gce was not checked here, which is breaking currently, I'm not seeing `security_group` option in service catalog with `gce_central` provider

Reply:
There was no change here to which providers this applies to, so if it was breaking for GCE on including security_group it must have been failing before this PR as well.